azure-kusto-data/tests/e2e.py

```diff
@@ -104,8 +104,15 @@ def setup_class(cls):
         cls.engine_cs = os.environ.get("ENGINE_CONNECTION_STRING") or ""
         cls.ai_engine_cs = os.environ.get("APPLICATION_INSIGHTS_ENGINE_CONNECTION_STRING") or ""
         cls.app_id = os.environ.get("APP_ID")
+        if cls.app_id:
```

Comment:
Why not use these (AZURE_CLIENT_ID ..) to begin with

Reply:
Not have to reconfigure github/anything else that uses the setup.